deployment/ds-hostnet-split/03-contour.yaml

```diff
@@ -36,9 +36,9 @@ spec:
         - --xds-port
         - $(CONTOUR_SERVICE_PORT)
         - --envoy-service-http-port
```

Comment:
Can you please make match troubleshooting.md and the help text generated by `contour serve -h`
```
- --envoy-service-http-port=80
- --envoy-service-https-port=443
```
Then we shouldn't have to deal with wacky yaml quoting rules

Reply:
Great point @dfc! I updated the other args to match so we don't have different examples.